lms/templates/discovery/course_card_2017.underscore

```diff
@@ -1,24 +0,0 @@
-<article class="course" role="region" aria-label="<%- content.display_name %>">
```

Comment:
@AlasdairSwan @MatthewPiatetsky do either of you know how this template got caught up in this? We don't use it for the programs pages and I'm confused about why it was touched. My intent is to leave this file deleted and revert changes to `lms/templates/discovery/course_card.underscore`.